docs/data-collection/pre-session.md

```diff
@@ -29,14 +29,11 @@ Instructions of operations to be performed before the participant arrival, **bef
 - [ ] Open psychopy 3 by typing `psychopy`
 - [ ] Load in the different experiments and check for proper functioning:
     - [ ] {{ settings.psychopy.tasks.func_rest }} (resting-state fMRI):
-        - [ ] time it to [confirm the length](intro.md#task-timing), and
```

Comment:
I feel we should confirm the length before the session. We are not doing it because we have other priorities, but I believe this is something we want to keep track of every time there are changes to the tasks.